unity-client/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -191,78 +191,7 @@ public class OnClickEventPayload
         public class SendChatMessageEvent
         {
             public ChatMessage message;
-        }
-
-        [System.Serializable]
-        public class AddEntityEvent
```

Comment:
Beware! Some of these payloads already exist in `Protocol.cs`. Please make sure we reuse the existing payloads if possible.

Also, in any case it would be nice to have them in a separate file, so the bridge class doesn't get overblown.

Reply:
After taking a look, the only payload that I can reuse is `RemoveEntity` payload, I will create another class with the info. However, I agree that we should have a single way to communicate with kernel